src/main/webapp/WEB-INF/web.xml

```diff
@@ -4,9 +4,23 @@
     xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd"
     version="3.0">
 
+    <listener>
+        <listener-class>ch.qos.logback.classic.servlet.LogbackServletContextListener</listener-class>
+    </listener>
+
     <listener>
         <listener-class>org.springframework.web.context.ContextLoaderListener</listener-class>
     </listener>
+
+    <listener>
+        <listener-class>org.terasoluna.gfw.web.logging.HttpSessionEventLoggingListener</listener-class>
+    </listener>
+
+    <context-param>
+        <param-name>logbackDisableServletContainerInitializer</param-name>
+        <param-value>true</param-value>
+    </context-param>
```

Comment:
- [ ] It's very bad. This fix obscures the effect of `contextConfigLocation`, so I recommend modifying it as follows:

```
context-param:logbackDisableServletContainerInitializer (add)
listener:LogbackServletContextListener (add)
listener:ContextLoaderListener + context-param:contextConfigLocation (no edit)
listener:HttpSessionEventLoggingListener (no edit)
```

This order is based on the following policy:

* `context-param` affects the entire application.
* Define `context-param` at the top that is not related to a particular listener.
* Define `context-param` associated with a particular listener immediately after the listener to clarify the relationship.

Reply:
Fixed it with https://github.com/terasolunaorg/terasoluna-gfw-web-blank/pull/478 .